docs/source/usage/config/log.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Managing CLI Output"
+sidebar_title: "Managing CLI Output"
+description: "Managing the information output by Rover"
+---
+
+Every Rover command takes an optional `--log/-l` argument that specifies the verbosity level of the tool. By default, Rover sets the log level to `info` which will print error, warning, and info logs directly to `stderr`. Setting the log level to `debug` will add some more information to the CLI output that is helfpul for debugging. Setting the log level to `trace` will make Rover log at the highest verbosity level. This is unnecessary for most use cases but can be helpful in some scenarios.
```

Comment:
Looking good! We use present tense instead of future tense pretty much everywhere we can:

"will print error" -> "prints error", "will add some more" -> "adds some more", etc.

Same meaning, one word shorter 😄